### DISCUSSION TITLE
Is there a way to exclude fcitx5 on certain applications?

### DISCUSSION THREAD

**Discussion by directentis1** [fcitx/fcitx5#1150] [Q&A]
I use Xfce4 and want to ask if it is possible to turn off/switch disable on some applications? As with Emacs, which uses its own IME, or with the Terminal Emulator, using another input method is not necessary. 

I tried searching but there were no specific instructions.

There is a suggestion from [Disable fcitx5 IME for a certain program in wayland. : linuxquestions](https://old.reddit.com/r/linuxquestions/comments/rlpzbk/disable_fcitx5_ime_for_a_certain_program_in/) is to unset variables like XMODIFIERS, GTK_IM_MODULE, QT_IM_MODULE, but it doesn't work for me.

**Comment by shadowe1ite** [comment]
Did you find anything?

**Comment by c02y** [comment]
This issue drives me crazy after I switched to Wayland, maybe this is a Wayland issue.

**Reply by shadowe1ite** [reply]
> This issue drives me crazy after I switched to Wayland, maybe this is a Wayland issue.

I never tried this, but i think it will work if your using Hyprland 

https://hyprland-community.github.io/pyprland/fcitx5_switcher

**Reply by c02y** [reply]
> > This issue drives me crazy after I switched to Wayland, maybe this is a Wayland issue.
> 
> I never tried this, but i think it will work if your using Hyprland
> 
> [hyprland-community.github.io/pyprland/fcitx5_switcher](https://hyprland-community.github.io/pyprland/fcitx5_switcher)

Thanks for the link, I'm using Hyprland, but I don't/wouldn't use pyprland.
